modules/tracing/mule-tracer-configuration-impl/src/main/java/org/mule/runtime/tracing/level/impl/config/FileTracingLevelConfiguration.java

```diff
@@ -203,31 +194,17 @@ private JsonNode getTracingLevelConfiguration() {
       if (parseBoolean(spanExporterConfiguration.getStringValue(MULE_OPEN_TELEMETRY_EXPORTER_ENABLED, "false"))) {
         if (LOGGER.isDebugEnabled()) {
           LOGGER
-              .debug(format("Non existent or non parseable tracing level config file found. The tracing level will be set to the default: %s",
-                            defaultLevel),
+              .debug(format("Non existent or non parseable tracing level config file found. The tracing level will be set to the default."),
```

Comment:
This is the part where you need to move the default to the AutoConfigurable class only. This one (The FileConfiguration) will no longer know about a default. It's the AutoConfigurable one the one that handles the default when the File one has failed (because of a typo in the config) or returned null. This is the class where you need to do a bit of refactor. Also, the message must not change, try not to loose the default value.

Also, take a look at the getTracingLevel() method because you will need to start checking the raw config value instead of the parsed value there (you no longer have the default value so the raw, file configured value is the only one you can strore).